.github/workflows/ci-pr.yml

```diff
@@ -44,6 +45,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Build
+      # ???
```

Comment:
See also: https://github.com/sorawee/racket/pull/4/files#r879175580

Reply:
Checkout with fetch-depth makes sense I think.

Reply:
Duplicate of the above question, so let me close this.